tests/event-bus/Gopkg.toml

```diff
@@ -12,6 +12,10 @@ required = ["k8s.io/code-generator/cmd/client-gen"]
   name = "k8s.io/client-go"
   version = "kubernetes-1.10.1"
 
-  [[constraint]]
+[[constraint]]
   name = "github.com/satori/go.uuid"
   version = "1.2.0"
+
+[prune]
```

Comment:
Please remove this section. We need to get the non-go files for K8S client code generation

Reply:
Updated `required` section where needed. Thank for pointing it.